### PR TITLE
✨ init extensionCloseCall

### PR DESCRIPTION
**Pull request by Cr4zyc4k3**
Should port this script https://github.com/Cr4zyc4k3/LSS/raw/master/LSS_expansion_close_radio_call.user.js into LSSM.
Should close extension radio requests, after click on "to mission".
Hopefully I didnt miss anything.

### CODE REVIEW

**Review by Suchty112 — APPROVED** [review]
Seems to work. Cool

File: src/modules/generalExtensions/assets/extensionCloseCall.ts [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+                return;
+            }
+            prev.classList.add('radio_message_close');
+            prev.setAttribute('vehicle_id', vid);
+        }
+    }
+    if (window.location.pathname == '/' && observertarget != null) {
+        observer.observe(observertarget, observerOptions);
+    }
+};
```
Comment (jxn-30) [inline src/modules/generalExtensions/assets/extensionCloseCall.ts:23]: Having an click eventListener on `#radio_messages_important` would be easier. You could check if the target was a "to this mission" button :)

Also saves performance as we do not need a mutationObserver then :)

File: src/modules/generalExtensions/main.ts [hunk 2]
```diff
@@ -61,6 +61,16 @@ export default (async (LSSM, MODULE_ID, $m) => {
             )
         ).default(LSSM, mapUndo, ownMapMarkers, getSetting, MODULE_ID);
     }
+    if (
+        window.location.pathname === '/' &&
+        (await getSetting<boolean>('extensionCloseCall'))
+    ) {
+        await (
```
Comment (jxn-30) [inline src/modules/generalExtensions/main.ts:68]: The asset is not async so the await is optional. Just as a side note for you, don't worry about it 😉

**Review by jxn-30 — CHANGES_REQUESTED** [review]

File: src/modules/generalExtensions/assets/extensionCloseCall.ts (hunk 1 above)
Comment (Cr4zyc4k3) [reply src/modules/generalExtensions/assets/extensionCloseCall.ts]: should be changed with 
https://github.com/LSS-Manager/LSSM-V.4/pull/432/commits/0f195102ecccf35198bcc1b8a2560b073f2d87a4

**Review by jxn-30 — APPROVED** [review]

**Review by KokosnussDEV — APPROVED** [review]

**Pull request merged**